ultravox/model/ultravox_model.py

```diff
@@ -56,6 +56,7 @@ def __init__(self, config: UltravoxConfig):
 
         self.loss_config = LossConfig()
         self.post_init()
+        self.multi_modal_projector.to(dtype=config.torch_dtype)
```

Comment:
Right now, refactoring a function for just two lines of code seems a bit overkill to me. ZQ will likely create such a function with CFormer PR anyway, so maybe better to keep it out to avoid it getting removed in the merge?.

Reply:
maybe? Seems like it's also easy to merge a 2-line function, but I don't feel super strongly about this

Reply:
Done.